frontend/web/components/TestWebhook.js

```diff
@@ -23,7 +23,7 @@ export default class TheComponent extends Component {
           })
           .catch((e) => {
               if (e.text) {
-                  e.text().then(error => this.setState({ error: `The server returned an error: ${error}` }));
+                  e.text().then(error => this.setState({ error: `The server returned an error: ${error}`, loading:false }));
```

Comment:
When there was a webhook error, it wasn't clearing the loading state which meant the test webhook button remained disabled.